website/content/partials/API_ENDPOINTS.tsv

```diff
@@ -0,0 +1,184 @@
+Name	API Endpoint	API Method	API Docs	Equiv Command	Command URL	ACL Required	Info Revealed with No Token	Additional Notes

```

Comment:
@markan, @luoxuan00733 : we could potentially even link to this .tsv file if someone wanted access to a raw spreadsheet (rather than using our interface) as a sort of workaround to meet their own needs. Not sure this is useful, just wanted to write it down as an idea to consider before I forgot.

Reply:
After working with this for a bit, I honestly am not sure if it's valuable to commit this or not; it's a pain to edit and read. Wondering if some sort of more structured format (JSON anyone?) would be more useful.